cli/spec/kontena/cli/vault/update_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_relative "../../../spec_helper"
+require 'kontena/cli/vault/update_command'
+
+describe Kontena::Cli::Vault::UpdateCommand do
+
+  include RequirementsHelper
+  include ClientHelpers
+
+  let(:subject) do
```

Comment:
The `ClientHelpers` should already do all of this boilerplate, down to the `allow(subject).to receive(:current_grid).and_return('test-grid')`?